Locker/Helpers/Devices.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Devices.swift
+//  Locker
+//
+//  Created by Zvonimir Medak on 20.10.2021..
+//
+
+import Foundation
+
+class Devices {
+    static let shared = Devices()
+
+    private(set) var devicesURL: URL? = Constants.devicesURL
+
+    private init() {}
+
+    func fetchDevices() {
+        guard let devicesURL = devicesURL else {
+            return
+        }
+        URLSession.shared.dataTask(with: devicesURL) { data, _, error in
+            guard let data = data, error == nil,
+                  let urlPath = BundleHelpers.bundleResource?
+                    .url(forResource: "devices", withExtension: "json") else { return }
```

Comment:
```suggestion
            guard let data = data, 
                  error == nil,
                  let urlPath = BundleHelpers.bundleResource?.url(forResource: "DeviceList", withExtension: "json")
            else { return }
```
Consider renaming the file.

Reply:
Change the JSON file name to BiometryAvailabilityDeviceList